zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,94 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use zebra_chain::parameters::Network;
+
+/// Whatever
+pub struct Params(pub Network);
+
+impl Params {
+    /// COIN
+    pub const COIN: u32 = 100000000;
```

Comment:
Consider moving this constant next to `MAX_MONEY`, then use it to calculate `MAX_MONEY`.

Also, write large numbers using separate groups of thousands:
```suggestion
    pub const COIN: i64 = 100_000_000;
```
https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/amount.rs#L247

Reply:
https://github.com/ZcashFoundation/zebra/pull/1051/commits/fa7d722886c9dbd2f65a48c348158c23ec842fa8